core/src/main/java/io/questdb/cutlass/http/processors/SettingsProcessor.java

```diff
@@ -32,29 +32,47 @@
 import io.questdb.network.PeerDisconnectedException;
 import io.questdb.network.PeerIsSlowToReadException;
 import io.questdb.std.CharSequenceObjHashMap;
+import io.questdb.std.Chars;
 import io.questdb.std.ObjList;
+import io.questdb.std.str.Utf8Sequence;
+import io.questdb.std.str.Utf8String;
 import io.questdb.std.str.Utf8StringSink;
+import io.questdb.std.str.Utf8s;
 
 import java.net.HttpURLConnection;
 
 public class SettingsProcessor implements HttpRequestProcessor {
+    private static final Utf8String V2 = new Utf8String("v2");
```

Comment:
Do we need versioning? I thought this version is not yet released.

Reply:
good point, let me remove versioning

Reply:
done